### ISSUE TITLE
Various UX bugs on Settings

### ISSUE THREAD

**Issue by lisasy** [forem/forem#11621]
**Describe the bug**

(Copied and pasted from Slack message with @ludwiczakpawel)

- save button doesn’t work (that’s probably pretty emergency)
- multiple save buttons being visible on /customization
- when you update any field, the main “save” button appears at the bottom of the screen. but it doesn’t work now with some of the controls.
- when i save certain things on /extensions there are some unexpected behaviors… sometimes it redirects me to /profile and sometimes it redirects me to some other places.

**Screenshots**
https://share.getcloudapp.com/BluQ0pQm

**Comment by rhymes** [comment]
@lisasy the first three have been addressed by https://github.com/forem/forem/pull/11615 and https://github.com/forem/forem/pull/11617 but they are yet to be deployed because we're having troubles deploying anything right now

**Comment by rhymes** [comment]
@lisasy the changes are online now, you can check on https://dev.to/settings/customization

**Comment by ludwiczakpawel** [comment]
Can be closed? Or the 4th bullet point is still unsolved?

**Comment by lisasy** [comment]
4th bulletpoint seems to still be unresolved :|

**Comment by rhymes** [comment]
@lisasy could you record the screen while you reproduce it? Thank you!

**Comment by ludwiczakpawel** [comment]
@rhymes Sometimes it takes me to /profile, sometimes it keeps me on /extensions and sometimes it takes me to completely different place (see when saving templates)

Video: https://d.pr/v/CoU41b

**Comment by rhymes** [comment]
@ludwiczakpawel yeah, those definitely need to be adjusted to always go to the proper page, thanks for the video! BTW you can upload inline videos to GitHub now

**Comment by ludwiczakpawel** [comment]
> you can upload inline videos to GitHub now

max 10mb 🤷

**Comment by mikeydorje** [comment]
Closing as stale. No activity since 2021.

If this issue persists on current DEV or your instance, please reopen with updated details/reproduction steps.
